docs/quick-reference/glossary.md

```diff
@@ -22,7 +22,7 @@ FASTA format
   A text-based format for representing nucleotide or amino acid sequences.
 
 octadecyl (C18)
-  An alkyl radical C(18)H(37) derived from an octadecane by removal of one hydrogen atom.
+  An alkyl radical :math:`C_{18}H_{37}` derived from an octadecane by removal of one hydrogen atom.
```

Comment:
Do you have a screenshot of this? From my experience, this will treat C and H as mathematical variables, which is not what we want.
Better to use :sub: for subscripts.

Reply:
Yes, just tested and it is not correct.
Please use :sub: for stoichiometric coefficients and :sup: for isotopes instead.